apps/full-site-editing/full-site-editing-plugin/full-site-editing/store/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import * as selectors from './selectors';
+
+registerStore( 'a8c/full-site-editing', {
```

Comment:
I'll leave this as a follow up exercise (non-blocking), but I think it'd be a good idea to add related unit tests for the new selector here.